index.html

```diff
@@ -1675,9 +1616,13 @@ <h4>Time expressions</h4>
       </section> <!-- time-constraints-section -->
 
       <section>
-        <h4>Layout</h4>
+        <h4>Layout and styles</h4>
         <p>This specification does not put additional constraints on the layout and rendering <a>features</a> defined in [[!TTML-IMSC1.2]].</p>
         <div class="note">Layout of the paragraphs may rely on the default TTML region (i.e. if no <code>&lt;layout&gt;</code> element is used in the <code>&lt;head&gt;</code> element) or may be explicit by the use of the <code>region</code> attribute, to refer to a <code>&lt;region&gt;</code> element present at <code>/tt/head/layout/region</code>.</div>
+
+        <p>Style references or inline styles MAY be used, using <code>style</code> attributes 
+          and/or <code>&lt;style&gt;</code> elements as defined in [TTML2] or [TTML-IMSC1.2].</p>
```

Comment:
Need to describe how to do inline styles too, e.g.:

```suggestion
        <p>Style references or inline styles MAY be used, using any combination of
        <code>style</code> attributes, 
        <code>&lt;style&gt;</code> elements and
        inline style attributes as defined in [TTML2] or [TTML-IMSC1.2].</p>
```